src/Filesystem/Folder.php

```diff
@@ -354,12 +354,28 @@ public static function isRegisteredStreamWrapper($path)
      *
      * @param string $path Path to check
      * @return string Set of slashes ("\\" or "/")
+     *
+     * @deprecated 3.6.5 This method will be removed in 4.0.0. Use correctSlashFor() instead.
```

Comment:
Deprecations should emit runtime warnings with `deprecationWarning()` in addition to the documentation tag.